presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -1612,6 +1612,10 @@ private static boolean isDistinct(SqlBaseParser.SetQuantifierContext setQuantifi
     private static String getType(SqlBaseParser.TypeContext type)
     {
         if (type.baseType() != null) {
+            if (type.baseType().DOUBLE_PRECISION() != null) {
+                // Make DOUBLE PRECISION alias for DOUBLE.
```

Comment:
Add a comment like this:

`// TODO: this is a temporary hack that should be removed when the new planner is introduced.`

(@martint is currently working revamping our new planner. It will take a while though.)